pandas/tests/test_errors.py

```diff
@@ -11,33 +11,38 @@
 @pytest.mark.parametrize(
     "exc",
     [
-        "UnsupportedFunctionCall",
-        "UnsortedIndexError",
-        "OutOfBoundsDatetime",
-        "ParserError",
-        "PerformanceWarning",
+        "AttributeConflictWarning",
```

Comment:
Guessing this is just alphabetized correct?

Reply:
I meant to call that out in the PR. But, yeah, I just alphabetized them